pyproject.toml

```diff
@@ -0,0 +1,168 @@
+# https://packaging.python.org/en/latest/specifications/pyproject-toml/
+[build-system]
+requires = ["pdm-backend", "wheel>=0.42"]
+build-backend = "pdm.backend"
+
+[project]
+name = "pymediainfo"
+description = "A Python wrapper for the mediainfo library."
+authors = [
+    {name = "Louis Sautier", email = "sautier.louis@gmail.com"},
+]
+readme = "README.rst"
+license = {text = "MIT"}
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: Implementation :: PyPy",
+]
+dynamic = ["version"]
+requires-python = ">=3.9"
+dependencies = [
+]
+
+[project.optional-dependencies]
+tests = [
+    "pytest>=6",
+    "pytest-cov",
+    "pytest-xdist",
+]
+docs = [
+    "alabaster",
+    "setuptools_scm",
+    "sphinx",
+]
+dev = [
+    "ipython",
+    "mypy>=1.0",
+    "black",
+    "isort",
+    "flake8",
+    "pylint",
+]
+
+[project.urls]
+Homepage = "https://github.com/sbraz/pymediainfo"
+Documentation = "https://pymediainfo.readthedocs.io/"
+Bugs = "https://github.com/sbraz/pymediainfo/issues"
+
+
+# https://pdm-project.org/latest/
+[tool.pdm.version]
+source = "scm"
+
+[tool.pdm.dev-dependencies]
+download_library = ["wheel>=0.44", "requests", "tqdm"]
+
+[tool.pdm.build]
+source-includes = ["scripts/", "tests/"]
+
+[tool.pdm.scripts.types]
+help = "Check type hints"
+cmd = "mypy --install-types --non-interactive --config-file=pyproject.toml {args:src tests}"
+
+[tool.pdm.scripts.docs]
+help = "Build and test documentation"
+composite = [
+  "sphinx-build -W --keep-going --color -b html docs docs/_build",
+  "sphinx-build -W --keep-going --color -b linkcheck docs docs/_build",
+#  "sphinx-build -W --keep-going --color -b doctest docs docs/_build",
+]
+
+[tool.pdm.scripts.test-nocov]
+help = "Run tests without coverage"
+cmd = "pytest {args:-n auto}"
+
+[tool.pdm.scripts.test]
+help = "Run tests with coverage"
+cmd = "pytest --cov --cov-report=term-missing --cov-config=pyproject.toml {args:-n auto}"
+
+[tool.pdm.scripts.download_library]
+help = "Download mediainfo library"
+cmd = "python scripts/download_library.py {args:-c --auto}"
+
+[tool.pdm.scripts.clean_library]
+help = "Clean mediainfo library"
+cmd = "python scripts/download_library.py -c"
+
+[tool.pdm.scripts.tag_wheel]
+help = "Tag the wheel for a specific platform"
+shell = "python -m wheel tags --remove --platform-tag={args} dist/*-py3-none-any.whl 2> /dev/null"
```

Comment:
Need to figure out how to make this work on Windows.

Reply:
I created a script to run this, it's overkill but it's cross-platform. See https://github.com/sbraz/pymediainfo/commit/c40c6833ea394004f33210b87256ae312f1b8331#diff-37f736c88fae9d1631b6b1e93e3eb6a250ddd4189eb6ea33f638f71407580247.

Reply:
I didn't think to make it cross-platform, better with a script then.